fury/actors/tests/test_peak.py

```diff
@@ -145,6 +144,37 @@ def test__points_to_vtk_cells():
     npt.assert_array_equal(actual, desired)
     npt.assert_equal(vtk_cells.GetNumberOfCells(), 3)
 
+def test_symmetric_param(interactive=False):
```

Comment:
Add a second blank line between functions to avoid `PEP 8: E302 expected 2 blank lines, found 1`.